torchtune/modules/attention_utils.py

```diff
@@ -22,7 +22,18 @@
         flex_attention,
     )
 
-    flex_attention_compiled = torch.compile(flex_attention, dynamic=False)
+    try:
+        flex_attention_compiled = torch.compile(
+            torch.compile(flex_attention, dynamic=False)
+        )
+    except:
+        # If user's torch version is <torch-2.7.0.dev20250205, compiling may fail.
+        # using max-autotune fixes this issue. Context: https://github.com/pytorch/torchtune/issues/2113
+        # TODO: deprecate me once 2.7 becomes stable.
```

Comment:
For what it's worth it may just fail period on other hardware regardless of PyTorch version (unless the issue of devices with low compute capability has been fixed as well)